docs/community/index.rst

```diff
@@ -5,7 +5,8 @@ Community
 Pyfar is completely open source and it’s license allows unrestricted
 educational and commercial use. It is made by the acoustics research
 community for the acoustics research community. We welcome any contributions
-– go ahead if you are in the mood.
+– go ahead if you are in the mood. For this, please note and follow our
+`code of conduct`_.
```

Comment:
Last meeting we decided to do it in this pr, we discussed to remove the link Here and add both links in the box as it was proposed by previous suggestions.